pkg/server/test/list_objects.go

```diff
@@ -48,7 +57,7 @@ type listObjectsTestCase struct {
 	useCheckCache          bool
 }
 
-func TestListObjectsRespectsMaxResults(t *testing.T, ds storage.OpenFGADatastore) {
+func TestListObjects(t *testing.T, ds storage.OpenFGADatastore) {
```

Comment:
Why change the test name? The assertions haven't changed

Reply:
The only test cases that existed herein before were ones specific to result limit checking. That is no longer the case. The test is more generic to generally testing ListObjects, and so that was the motive behind the name change.

Reply:
I would prefer that we keep all the correctness tests in `consolidated_1_1_tests.yaml`

Reply:
I would too, but we don't have ABAC DSL models in the scope of this PR 😄